packages/wasm-edit/src/apply.js

```diff
@@ -192,19 +194,37 @@ function applyAdd(ast: Program, uint8Buffer: Uint8Array, node: Node): State {
   const newByteArray = encodeNode(node);
 
   // start at the end of the section
-  const start = sectionMetadata.startOffset + sectionMetadata.size.value + 1;
+  // the section size doesn't include the encoded u32 length (of the actual size)
+  // and of the vector size, we need to add them to the section size.
+  const u32EncodedLengths =
+    sectionMetadata.size.loc.end.column -
```

Comment:
braces makes sense here? adding the difference right?

Reply:
Yeah but Prettier removed them but math is still fine